lib/testmap.py

```diff
@@ -153,14 +140,15 @@
     },
     'candlepin/subscription-manager': {
         'main': [
-            'rhel-8-4',
```

Comment:
please leave a RHEL 8 for `main`: even if targets RHEL 9, work done in it is backported to RHEL 8 branches when needed; hence, it would be good to still make sure RHEL 8 works

Reply:
So for main you want to test 8.4, 8.5 and 9.0?